### PR TITLE
Add CloneRoute testcase to RouteSRV

### PR DESCRIPTION
**Pull request by MustafaSaber**
Follow up on https://github.com/zalando/skipper/pull/2131

Signed-off-by: Mustafa Abdelrahman <mustafa.abdelrahman@zalando.de>

### CODE REVIEW

File: routesrv/routesrv_test.go [hunk 1]
```diff
@@ -522,3 +523,63 @@ func TestESkipBytesHandlerWithXCount(t *testing.T) {
...
+		EditRoute: []*eskip.Editor{
+			eskip.NewEditor(regexp.MustCompile("Host[(](.*)[)]"), "HostAny($1)"),
+		},
+	})
+
+	rs.StartUpdates()
+	if err := tl.WaitFor(routesrv.LogRoutesInitialized, waitTimeout); err != nil {
+		t.Error("routes not initialized")
+	}
+	w := getRoutes(rs)
```
Comment (szuecs) [inline routesrv/routesrv_test.go:544]: w? Why not `r`?

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Comment by MustafaSaber** [comment]
👍

**Pull request merged**